tests/draft2019-09/unevaluatedItems.json

```diff
@@ -480,6 +480,50 @@
             }
         ]
     },
+    {
+        "description": "unevaluatedItems with $recursiveRef",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/2019-09/schema",
+            "$id": "https://example.com/extended-tree",
+
+            "$recursiveAnchor": true,
+
+            "$ref": "/tree",
+            "prefixItems": [
+                true,
+                true,
+                { "type": "string" }
+            ],
+
+            "$defs": {
+                "tree": {
+                    "$id": "/tree",
+                    "$recursiveAnchor": true,
+
+                    "type": "array",
+                    "prefixItems": [
```

Comment:
`prefixItems` is 2020-12.

Reply:
Good catch. I seems I accidentally have `prefixItems` registered as a 2019-09 keyword, so the test passed when I tested on my implementation.